fvm/src/gas/price_list.rs

```diff
@@ -648,3 +634,14 @@ pub fn price_list_by_network_version(network_version: NetworkVersion) -> &'stati
         _ => &SKYR_PRICES,
     }
 }
+
+impl Rules for &WasmGasPrices {
+    fn instruction_cost(&self, _instruction: &Instruction) -> Option<u64> {
+        Some(self.exec_instruction_cost_milli)
```

Comment:
There are a few wasm instructions we don't charge for: https://github.com/filecoin-project/FIPs/pull/368/files#diff-399066a5694eb92e4ac1648a75e6e683f9fc26afce520e5b5e17d6131cf695d5R109-R110

Should this handle the filtering?

Reply:
Need to look into this more, but those instructions are 'structural', so we may already not be charging for those.